main.go

```diff
@@ -2,18 +2,19 @@ package main
 
 import (
 	"fmt"
-	"path/filepath"
-
 	"github.com/bilginyuksel/cordova-plugin-helper/generator"
 	"github.com/bilginyuksel/cordova-plugin-helper/parser"
 	"github.com/bilginyuksel/cordova-plugin-helper/reader"
 	"github.com/bilginyuksel/cordova-plugin-helper/writer"
+	"path/filepath"
 
 	"github.com/alecthomas/kong"
 )
 
 func main() {
-	prepareCliParser()
+	//prepareCliParser()
```

Comment:
I think you have commented this line because you were testing the application. But when you finished testing I see that you forgot to uncomment it. 😄